test/unit/org/apache/cassandra/io/sstable/LegacySSTableTest.java

```diff
@@ -101,7 +101,7 @@ public class LegacySSTableTest
     // Get all versions up to the current one. Useful for testing in compatibility mode C18301
     private static String[] getValidLegacyVersions()
     {
-        String[] versions = {"oa", "da", "nc", "nb", "na", "me", "md", "mc", "mb", "ma"};
+        String[] versions = {"oa", "da", "nb", "na", "me", "md", "mc", "mb", "ma"};
         return Arrays.stream(versions).filter((v) -> v.compareTo(BigFormat.getInstance().getLatestVersion().toString()) <= 0).toArray(String[]::new);
```

Comment:
btw. why is this filtering? It seems like it does not filter anything. Moreover if feels incorrect as it assumes `BigFormat`

Reply:
`oa` needs to be filtered out unless you do `test-oa`. So in compatibility mode you can go up to nb, out of it you can go up to oa.

Reply:
so it is just because we cannot read oa sstables